### DISCUSSION TITLE
How to set error in field, when field is set validate？

### DISCUSSION THREAD

**Discussion by walkonairy** [react-hook-form/react-hook-form#7365] [Q&A]
Hello！I have an Input and I define a validate. I called the setError event when it triggered the Blur event. I want to show the error message set in setError directly, rather than executing the validate method again。What should I do, please。

**Comment by bluebill1049** [comment]
https://react-hook-form.com/api/useform/seterror

refer to the doc 

```
{ type: string, message?: string, types: MultipleFieldErrors }
```

```
<input
    onChange={(e) => {
      setError("username", {
        type: "manual",
        message: "Dont Forget Your Username Should Be Cool!",
      });
      onChange(e);
    }}
    {...rest}
  />
```
